components/scream/src/mct_coupling/scream_scorpio_interface.F90

```diff
@@ -415,6 +445,8 @@ subroutine eam_init_pio_subsystem(mpicom,atm_id,local)
     integer :: num_aggregator
     integer :: base
 
+    if (associated(pio_subsystem)) call errorHandle("PIO ERROR: pio_subsystem has already been defined.",-999)
```

Comment:
Maybe I don't understand what 'sync-ing' and pio_subsystem mean, but...isn't the PIO subsystem expected to be already defined at when you try to sync a file?

Reply:
Ahh yes, this is the local pointer to the subsystem that has already been defined.  So it is really a matter of whether or not the module level pointer has been assigned.  Perhaps a better error message would be something like "PIO ERROR: The pio_subsystem pointer for this component has already been established", I'll give it some thought on a better message, that too may be a bit too generic.